modules/codegen/src/main/scala/com/twilio/guardrail/generators/Java/AsyncHttpClientClientGenerator.scala

```diff
@@ -471,18 +555,47 @@ object AsyncHttpClientClientGenerator {
 
           val callBuilderCls = new ClassOrInterfaceDeclaration(util.EnumSet.of(PUBLIC, STATIC), false, callBuilderName)
           callBuilderFinalFields.foreach({ case (tpe, name) => callBuilderCls.addField(tpe, name, PRIVATE, FINAL) })
+          val callBuilderInitContentType = consumes.map(
+            ct =>
+              new ExpressionStmt(
+                new MethodCallExpr(
+                  "withHeader",
+                  new StringLiteralExpr("Content-Type"),
+                  new StringLiteralExpr(ct.value)
+                )
+            )
+          )
+          val callBuilderInitAccept = {
+            val acceptHeaderString = produces
+              .flatMap({ case (_, ct) => ct.map(ct => s"${ct.value}; q=1.0") })
+              .mkString(", ")
+            if (acceptHeaderString.nonEmpty) {
+              Option(
+                new ExpressionStmt(
+                  new MethodCallExpr(
+                    "withHeader",
+                    new StringLiteralExpr("Accept"),
+                    new StringLiteralExpr(s"$acceptHeaderString, */*; q=0.1")
+                  )
+                )
+              )
+            } else {
+              Option.empty
+            }
+          }
 
           callBuilderCls
             .addConstructor(PRIVATE)
             .setParameters(callBuilderFinalFields.map({ case (tpe, name) => new Parameter(util.EnumSet.of(FINAL), tpe, new SimpleName(name)) }).toNodeList)
             .setBody(
               new BlockStmt(
-                callBuilderFinalFields
-                  .map[Statement, List[Statement]]({
-                    case (_, name) =>
-                      new ExpressionStmt(new AssignExpr(new FieldAccessExpr(new ThisExpr, name), new NameExpr(name), AssignExpr.Operator.ASSIGN))
-                  })
-                  .toNodeList
+                (
+                  callBuilderFinalFields
+                    .map[Statement, List[Statement]]({
```

Comment:
Is this just to avoid `List[Statement](new ExpressionStmt(..))`?

Reply:
No, it's already a list.  unfortunately mapping without the type ascription gives us a `List[ExpressionStmt]`, then `NodeList[ExpressionStmt]`, which, due to variance (or that lack thereof?), makes the types not line up, because the `BlockStmt` constructor wants a `NodeList[Statement]`.

Reply:
Ah, so this is to avoid `new ExpressionStmt(...): Statement`. Makes sense.